### DISCUSSION TITLE
Where is the apk version?

### DISCUSSION THREAD

**Discussion by ZarkTye** [diasurgical/DevilutionX#2795] [Q&A] [answered]
![Screenshot_20210904-231217](https://user-images.githubusercontent.com/81594191/132112691-1111955f-e5b1-46dd-9245-67e74d1cfd68.png)

**Comment by julealgon** [comment]
There is no Android 1.1.0 release.

**Comment by ZarkTye** [comment, marked as answer]
Ok

Em sáb, 4 de set de 2021 23:20, Juliano Leal Goncalves <
***@***.***> escreveu:

**Reply by julealgon** [reply]
If you want to experiment with the android version, you can compile it using the instructions in the main page. It should be coming in version 1.3.0.

**Comment by ZarkTye** [comment]
but this version is 1.2.1

Em sáb, 4 de set de 2021 23:20, Juliano Leal Goncalves <
***@***.***> escreveu:

**Reply by julealgon** [reply]
I saw the 1.1.0 on your screenshot and got confused for a second. Regardless, the official Android version is coming in 1.3.0. The version currently in the Google store is a beta version.

**Reply by StephenCWills** [reply]
Right now, we don't have any release versions for Android. We do have APKs for the test builds with physical gamepad support, or you can build it yourself as was previously mentioned. Touch controls are a work-in-progress. We plan to release a version via the Play Store before November.

**Comment by ZarkTye** [comment]
Brasil?

Em sáb, 4 de set de 2021 23:27, Juliano Leal Goncalves <
***@***.***> escreveu:

**Reply by julealgon** [reply]
If you are asking if I'm from Brazil, yes I am.

**Comment by ZarkTye** [comment]
Ok I'll wait

Em sáb, 4 de set de 2021 23:30, Stephen C. Wills ***@***.***>
escreveu:

**Comment by ZarkTye** [comment]
Yes me too

Em dom, 5 de set de 2021 00:08, Juliano Leal Goncalves <
***@***.***> escreveu:

**Reply by AJenbo** [reply]
Feel free to help finish the Portuguese translation.
